.node-version

```diff
@@ -0,0 +1 @@
+5.7.1
```

Comment:
Ah, ok. Personally I use `nvm` for that purpose, so let's also add a `.nvmrc` :grin: